app/services/parameter_safe_render.rb

```diff
@@ -26,12 +30,14 @@ def render_object(object)
     when Hash
       object.merge(object) { |k, v| render_object v }
     else
-      # Don't know how to parse this, send it back
       object
     end
   end
 
   def render_string(string)
+    # exit early if there is nothing to parse (minimum possible ERB is four characters '<%%>')
+    return string if (string.nil? || string.size <= 4 || !string.index('<%'))
```

Comment:
we have `String#contains_erb?` in `lib/core_extentions.rb` - might be good to use that (and improve it if possible).

Reply:
Will do, our implementation is not the fastest :-)

Reply:
also, `string != nil` since this function is only called when the object is a `String`

Reply:
Well not anymore when I extract this into `contains_erb?` I guess.

Reply:
it is because `contains_erb?` is monkey patched to String class. So not crucial, but the nil check is redundant both in this function and in that one.

Reply:
Oh now I got what you mean, sure.